src/pds_doi_service/core/actions/check.py

```diff
@@ -103,11 +103,11 @@ def _update_transaction_db(self, pending_record):
 
         """
         doi_value = pending_record['doi']
-        lidvid = '::'.join([pending_record['lid'], pending_record['vid']])
+        identifier = pending_record['identifier']
```

Comment:
Thank you 🙏

Using `'::'.join` on a fixed 2-item list instead of just `a + '::' + b` (or even `f'{a}::{b}'` was driving me nuts! 🤪